instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld

```diff
@@ -692,20 +692,25 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Kleven et al. 2023.",

```

Comment:
The publications should be stated as full citations. you don't want to force people to look it up themselves.

Reply:
Full citation has been added.